app/views/publishers/_choose_channel_type.html.slim

```diff
@@ -38,3 +38,12 @@
         .icon-and-text--text
           h6= t "publishers.choose_new_channel_type.twitter.heading"
           = t "publishers.choose_new_channel_type.twitter.description"
+
+= link_to(publisher_register_vimeo_channel_omniauth_authorize_path, class: 'card block-link--card', :"data-piwik-action" => "AddVimeoChannelClicked", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "AddChannelFlow") do
+  .card-body
+    .icon-and-text--wrapper
+      .icon-and-text--icon
+        = image_tag "choose-channel/choose-channel--vimeo.jpg", height: 72, width: 88, class: "choose-channel--icon"
```

Comment:
I don't think we should specify a width because right now it's scaled incorrectly